### DISCUSSION TITLE
Friendly-snippets not working

### DISCUSSION THREAD

**Discussion by ChenZhongPu** [LazyVim/LazyVim#4921] [Q&A]
I enabled `LuaSnip` in lazyvim extra, but I found that the configurations in `snippets` does not take effect.

![image](https://github.com/user-attachments/assets/0fabcc89-97d5-4632-8d22-00360511cba2)

For example, the content of `markdown.json`  is:

```json
{
  "meta": {
    "prefix": "meta",
    "body": [
      "---",
      "title: ${1:note_title}",
      "",
      "author: My Name",
      "",
      "date: ${2:date}",
      "",
      "CJKmainfont: LXGW WenKai",
      "---"
    ],
    "description": "Meta for PDF output"
  }
}
```
But when I opened a Markdown file, the input `meta` doesn't trigger such code snippet. How to debug and fix it?

**Comment by folke** [comment]
luasnip requires an index file in that directry as well. Check the friendly snippets repo for an example

**Reply by ChenZhongPu** [reply]
Thanks. I also noticed that a `package.json` is required. So I added it inside `snippets` folder:

```json
{
  "name": "my-friendly-snippet",
  "contributes": {
    "snippets": [
      {
        "language": [
          "markdown"
        ],
        "path": "./markdown.json"
      },
      {
        "language": [
          "tex"
        ],
        "path": "./latex.json"
      }
    ]
  }
}
```

But it still cannot be triggered.

**Comment by ChenZhongPu** [comment]
Any update with this question?

**Comment by ChenZhongPu** [comment]
The bug was [fixed](https://github.com/LazyVim/LazyVim/pull/4987/files).
